### DISCUSSION TITLE
Model: fleet structure

### DISCUSSION THREAD

**Discussion by melissamonk-NOAA** [pfmc-assessments/CA_quillback_rockfish_2025#6] [General]
Discussion and questions related to possible fleet structure

1. Commercial
   a. Live fish fishery
   b. Dead fish fishery
   c. All gears combined?
   d. How many are discarded?  Are deceased fish from the live fish fishery sold dead?
   e. Go by sector (i.e. LE, catch shares) 

We have historical landings broken out by other and trawl (though most are trawl). 1916-1968 dont currently have across areas, but 1968-1980 does.

2. Recreational
   a. Private/rental
   b. CPFV (charter boat)
   c. Discards included in these?

We have historical landings broken out by PC and PR modes. Do not have broken out by area but probably could because it is available N/S Pt. Conception. 

3. Growth fleet 
    a. Do we need a ghost fleet to inform growth?

4. CCFRP survey fleet
5. ROV survey fleet

**Comment by brianlangseth-NOAA** [comment]
**With respect to recreational PC/PR mode**

I've been able to process the recreational length data and based on this analysis, PC/PR modes have very similar lengths when analyzed within the same areas. 

![recfin_length_mode_area_violin](https://github.com/user-attachments/assets/eaa9f183-f2da-408d-a337-0d8bb1561711)

Aggregating over all areas is deceptive, because lengths are larger in northern districts than in southern districts, and PC modes has a greater number of samples in the southern districts, thus PC mode when aggregated appears to catch smaller fish. 

![recfin_length_area_density](https://github.com/user-attachments/assets/344b97aa-2383-441e-9e05-86095b1006cd)

![recfin_length_mode](https://github.com/user-attachments/assets/cb1659a8-580b-49b0-854a-eed0222de958)

Based on this I wonder whether the greter size in northern districts is due to biology (are fish really that much bigger there) or is it an effect of the fishery (are they fishing different depths or areas)? These would be questiosn to ask fishers. Based on commercial data, northern ports have larger fish as well.  

As far as I can tell, fish are generally similarly sized over time though of course mean length plots show trends better than raw data. 

![recfin_length_area](https://github.com/user-attachments/assets/b4206d6d-39d0-47cf-971e-42f9b6f1e399)

**Comment by brianlangseth-NOAA** [comment]
**With respect to commercial live/dead fish fishery**

My explorations thus far in the PacFIN commercial data indicate little difference in sizes of fish with a disposition of "alive" versus not. This would suggest that there is not a sized based preference for live fish over dead fish. Lengths tend to overlap in years where live and dead co-occur, and the overall size is not greatly different. 

![pacfin_length_disposition](https://github.com/user-attachments/assets/340d2ff6-63af-46c7-9cdc-d545be8aed7a)

Nor are lengths for live/dead dependent on where fish are caught. Note that in the below figure Bragg lengths for live and dead are not overlapping in time and is the reason for seemingly different lengths between live and dead. 

![pacfin_length_port_group_disposition](https://github.com/user-attachments/assets/2d9afc2d-7464-425a-96cc-b95656ae30c5)

![pacfin_length_port_group_disposition_density](https://github.com/user-attachments/assets/37c2e21f-04aa-4b48-8730-092b8f5deb7c)
